### ISSUE TITLE
[Bug]: Cannot write to /Android/data causing error?

### ISSUE THREAD

**Issue by Omay238** [LemonLoader/MelonLoader_057#66]
### All of the following criteria must be met

- [X] Full ``Latest.log`` file included. If no file exists then leave this unchecked and state so.

### All of the following are optional to answer

- [X] Tried reinstalling the game.
- [X] Tried reinstalling LemonLoader.
- [X] Tried restarting device.

### Describe the issue.

[Latest.log](https://github.com/LemonLoader/MelonLoader/files/13358298/Latest.log)
After this part, the game crashes. I'm using BONELAB public beta, with Quest 3 on latest OS version as of November 14 2023. When patching with Lemon Loader 0.2.0, at the end, before the install step, it says that it can't write to /Android/data, and that may break some games, and when I press continue, then run BONELAB, the game crashes after this log. I allowed Lemon Installer access to all files, install apps from this source, access to /Android/data and /Android/obb, but it still gives this error.

### Did you attach your log file?

- [X] Yes, I attached my log file to the text box above.
- [ ] No, I could not find a log file at `/storage/emulated/0/Android/data/<package name>/files/melonloader/etc/Latest.log`

**Comment by batnight9** [comment]
can some one explain in simpler terms pls?

**Comment by batnight9** [comment]
i have reinstalled the game and lemon installer and restarted my device it still has the same error, any other answer?

**Comment by TrevTV** [comment]
I don't remember exactly when the update rolled out but possibly due to the new-ish Quest update. Either wait for Meta to fix it, or wait for me to maybe possibly do LemonLoader/MelonLoader#7
